pkg/virt-launcher/virtwrap/live-migration-source.go

```diff
@@ -382,11 +382,15 @@ func (l *LibvirtDomainManager) setMigrationResultHelper(failed bool, reason stri
 		return nil
 	}
 
+	metaAbortStatus := migrationMetadata.AbortStatus
 	if abortStatus != "" {
-		metaAbortStatus := migrationMetadata.AbortStatus
 		if metaAbortStatus == string(abortStatus) && metaAbortStatus == string(v1.MigrationAbortInProgress) {
 			return domainerrors.MigrationAbortInProgressError
 		}
+	} else if metaAbortStatus == string(v1.MigrationAbortInProgress) &&
```

Comment:
This branch will be taken only if `abortStatus` is empty or am I missing something?

Reply:
Fixed this, I made it simpler.